src/index.html

```diff
@@ -1,12 +1,336 @@
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo logo" >
+      </a>
+      <nav class="header__menu">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#Apple" class="header__link is-active">Apple</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Samsung" class="header__link">Samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Smartphones" class="header__link">Smartphones</a>
+          </li>
+
+          <li data-qa="nav-hover" class="header__item">
+            <a href="#Laptops" class="header__link">Laptops & Computers</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Gadgets" class="header__link">Gadgets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Tablets" class="header__link">Tablets</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Photo" class="header__link">Photo</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#Video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="card">
+        <div data-qa="card"  class="card__item">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="stars__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__cost">
+            <p class="card__name">Price:</p>
+            <p class="card__price">$2,199</p>
+          </div>
+
+          <a  href="#"
+           data-qa="card-hover"
+           class="card__button">
+            Buy
+          </a>
+        </div>
+        <div  class="card__item">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+
+          <h1 class="card__title">
```

Comment:
h1 tag is used for page titles. I think semantically it is better to not use it here